iib/workers/tasks/build.py

```diff
@@ -858,6 +859,11 @@ def handle_add_request(
     present_bundles_pull_spec = []
     with tempfile.TemporaryDirectory(prefix='iib-') as temp_dir:
         if from_index:
+            if is_image_dc(from_index):
+                warn_msg = 'Declarative config image type is not supported yet.'
+                log.warning(warn_msg)
```

Comment:
Do we need this log here?

Reply:
I do not see why not, it is present in different parts of IIB as well (log and raise).

Reply:
how does the log look like? Wouldn't be duplicated? Or do we have it for Splunk logging?

Reply:
@lipoja ?

Reply:
`log.warning` is for logging and raising exception is for stopping execution of code with not supported input. 
Logging can be redirected to different output. I believe `Traceback` is printed on `stderr` and is not directly connected to logging.

Reply:
thank you

Reply:
I found this as an good explanation of logs: https://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels

Reply:
Yeah, I guess we can go with error message.